windows_containers/wmco_rn/windows-containers-release-notes-10-17-x-prereqs.adoc

```diff
@@ -42,7 +49,7 @@ a|* Windows Server 2022, OS Build link:https://support.microsoft.com/en-us/topic
 
 == Supported networking
 
-Hybrid networking with OVN-Kubernetes is the only supported networking configuration. See the additional resources below for more information on this functionality. The following tables outline the type of networking configuration and Windows Server versions to use based on your platform. You must specify the network configuration when you install the cluster. 
+Hybrid networking with OVN-Kubernetes is the only supported networking configuration. See the additional resources below for more information on this functionality. The following tables outline the type of networking configuration and Windows Server versions to use based on your platform. You must specify the network configuration when you install the cluster.
```

Comment:
```suggestion
Hybrid networking with OVN-Kubernetes is the only supported networking configuration. The following tables outline the type of networking configuration and Windows Server versions to use based on your platform. You must specify the network configuration when you install the cluster.
```
The second sentence doesn't seem necessary. I'd assume that the tables and notes that follow this paragraph contain related information because they're all in the same section.

Reply:
It appears that there was an Addition Resources section at some point, likely linking to [Configuring hybrid networking](https://docs.openshift.com/container-platform/4.17/networking/ovn_kubernetes_network_provider/configuring-hybrid-networking.html). I replaced the link.